Floofbot/Handlers/CommandHandler.cs

```diff
@@ -61,6 +61,12 @@ private async Task HandleCommandAsync(SocketMessage s)
 
             if (msg.HasStringPrefix(prefix, ref argPos) || msg.HasMentionPrefix(_client.CurrentUser, ref argPos))
             {
+                if(!msg.HasStringPrefix(prefix, ref argPos)) {
```

Comment:
I think this should be `if msg.HasMentionPrefix(_client.CurrentUser, ref argPos)` instead to make it clearer that we're targeting mentions only.
Also, we might need to distinguish between responding to a question and running a command. 
For example, what if the command is `@Floofbot tag list example?`

Reply:
> I think this should be if msg.HasMentionPrefix(_client.CurrentUser, ref argPos) instead to make it clearer that we're targeting mentions only.
True, will fix

> Also, we might need to distinguish between responding to a question and running a command.
> For example, what if the command is @Floofbot tag list example?

You're right. I'm not sure how to fix this though...

Reply:
I'll check for `result.Error == CommandError.UnknownCommand` and fire the `QuestionEightBall` only if that happens. Should be robust enough?